CHANGELOG.md

```diff
@@ -1,5 +1,96 @@
+
+### Table Changes
+
+- Add ETW-based process events table for Windows ([#7821](https://github.com/osquery/osquery/pull/7821))
+- Add `pid_with_namespace` for `yara` table ([#7920](https://github.com/osquery/osquery/pull/7920))
+- Add a new table `kernel keys` to the Linux platform ([#7876](https://github.com/osquery/osquery/pull/7876))
+- Leave min_version empty in xprotect_meta when not specified ([#7926](https://github.com/osquery/osquery/pull/7926))
+- Port the `secureboot` table to macOS ([#7692](https://github.com/osquery/osquery/pull/7692))
+- Update `docker_container_stats` table to include `cached_memory` column ([#7807](https://github.com/osquery/osquery/pull/7807))
+- `cpu_info`: Port the table to macOS x86 and Apple Silicon ([#7757](https://github.com/osquery/osquery/pull/7757))
+- experiments: Implement a new bpf_process_events_v2 table ([#7773](https://github.com/osquery/osquery/pull/7773))
+- systemd_units: Add new unit_file_state column ([#7895](https://github.com/osquery/osquery/pull/7895))
+
+### Under the Hood improvements
+
+- Set counter consistently so zero always indicates all records ([#7801](https://github.com/osquery/osquery/pull/7801))
+- Support logging empty result set in batch format for initial runs ([#7803](https://github.com/osquery/osquery/pull/7803))
+- Support rollbacks of osquery when new versions introduce new column families ([#7712](https://github.com/osquery/osquery/pull/7712))
+- analysis.py: Add --pack flag to load queries from a pack file ([#7935](https://github.com/osquery/osquery/pull/7935))
+- profile.py: Log # of queries loaded and raise an error if 0 are loaded ([#7934](https://github.com/osquery/osquery/pull/7934))
+
+### Bug Fixes
+
+- Clear cached constraints and columns in xBestIndex ([#7435](https://github.com/osquery/osquery/pull/7435))
+- Fix assert fail for unverified WMI request result ([#7921](https://github.com/osquery/osquery/pull/7921))
+- Fix leaks in scheduled_tasks (#7903) ([#7904](https://github.com/osquery/osquery/pull/7904))
+- Flush console buffer during ungraceful exit ([#7829](https://github.com/osquery/osquery/pull/7829))
+- Propagate windows errors to the exit code ([#7896](https://github.com/osquery/osquery/pull/7896))
+- Relax osquery safe permissions check ([#7763](https://github.com/osquery/osquery/pull/7763))
+- Silence warnings for more builtin Chrome and Brave extensions ([#7932](https://github.com/osquery/osquery/pull/7932))
+- Workaround for hung routes table ([#7916](https://github.com/osquery/osquery/pull/7916))
+- dns_resolvers: fix typo in the name when spawning in namespace ([#7875](https://github.com/osquery/osquery/pull/7875))
+- test: Fix flaky test_daemon_sigint ([#7888](https://github.com/osquery/osquery/pull/7888))
+
+### Documentation
+
+- Add note about windows_security_products compatibility ([#7880](https://github.com/osquery/osquery/pull/7880))
+- CHANGELOG 5.7.0 ([#7894](https://github.com/osquery/osquery/pull/7894))
```

Comment:
```suggestion
```

Reply:
Eh? Why remove this?

Reply:
I don't see any reason why a user would benefit from seeing that the changelog was updated for 5.7 in the 5.8.1 release notes. I don't see much harm either though so if you want to leave it there that's fine with me.

Reply:
I agree it's of low value, but I believe in being complete about them. I've generally tended to including _all_ changes.